config/nightwatch.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
     },
     wcag2a: {
       globals: {
-        rules: ['section508', 'wcag2a']
+        rules: ['section508', 'wcag2a', 'wcag2aa']
```

Comment:
Listing both is recommended in the second example provided [here](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter-examples).